### DISCUSSION TITLE
kong.response.get_raw_body() does not work

### DISCUSSION THREAD

**Discussion by jianmaikj** [Kong/kong#8188] [Help] [answered]
```
function plugin:body_filter(conf)
    local resp_body = kong.response.get_raw_body()
    kong.log.set_serialize_value('resp_body', resp_body)
end
```
I want to get the response body and add it to the log, but get:`attempt to call field 'get_raw_body' (a nil value)`？

**Comment by flrgh** [comment, marked as answer]
Hi @jianmaikj :wave: 

The `kong.response.get_raw_body()` PDK method has been [merged in to master](https://github.com/Kong/kong/pull/7877) but not yet published in a release (note the "unreleased" [section of the changelog](https://github.com/Kong/kong/blob/a3249a8d0a53d54f7efab00a5de9d836957cb871/CHANGELOG.md#Unreleased)). It will be available in the next release (2.7) which is due _very_ soon.

**Reply by jianmaikj** [reply]
> Hi @jianmaikj 👋
> 
> The `kong.response.get_raw_body()` PDK method has been [merged in to master](https://github.com/Kong/kong/pull/7877) but not yet published in a release (note the "unreleased" [section of the changelog](https://github.com/Kong/kong/blob/a3249a8d0a53d54f7efab00a5de9d836957cb871/CHANGELOG.md#Unreleased)). It will be available in the next release (2.7) which is due _very_ soon.

I see,thanks!
